Boolti/Boolti/Sources/UILayer/Ticket/TicketDetail/Views/TicketNoticeView.swift

```diff
@@ -40,22 +44,22 @@ final class TicketNoticeView: UIView {
     private func configureUI() {
         self.addSubviews([
             self.titleLabel,
-            self.noticeLabel
+            self.noticeTextView
         ])
 
         self.titleLabel.snp.makeConstraints { make in
             make.top.equalToSuperview().inset(16)
             make.horizontalEdges.equalToSuperview()
         }
 
-        self.noticeLabel.snp.makeConstraints { make in
-            make.top.equalTo(self.titleLabel.snp.bottom).offset(12)
+        self.noticeTextView.snp.makeConstraints { make in
+            make.top.equalTo(self.titleLabel.snp.bottom).offset(10)
             make.horizontalEdges.equalTo(self.titleLabel)
             make.bottom.equalToSuperview().inset(20)
         }
     }
 
     func setData(with text: String) {
-        self.noticeLabel.text = text
+        self.noticeTextView.text = text
```

Comment:
여기 아마 textView에 텍스트 height 적용이 안되고 있을 거라서 행간 확인이 필요할 것 같아!!

```suggestion
        self.noticeTextView.text = text
        self.noticeTextView.setLineSpacing(lineSpacing: 6)
```